### ISSUE TITLE
Exporting Bookmarks

### ISSUE THREAD

**Issue by Yutolio** [minbrowser/min#818]
Hi!
I would like to export the bookmarks I have on Min but I can't find a way to do so.
Even just a list of all the bookmarked links that I can copy-paste would be fine

**Comment by PalmerAL** [comment]
There isn't a built-in way to do that; it should be fairly simple to do, so I'll try to add it soon. In the meantime, there's a somewhat more complicated way to get a list:

1. Open the developer tools (developer > inspect browser > console)
2. Click on "top" and choose "placesWorker.js", and paste this script:
```
var c = copy, items = []; db.places.each(item => {if (item.isBookmarked) {items.push(item.url)}}).then(() => {console.log("copied to clipboard");c(items.join("\n"))})
```

**Comment by PalmerAL** [comment]
Fixed in ee09ee0f6fd36836619ab2ee83681accc16f6077, starting in the next release there will be an `!exportbookmarks` command. Running that will give you an HTML file that you can import into other browsers.

This turned out to be a bit trickier than I expected, due to the fact that the standard format for a bookmarks file comes from Netscape (http://fileformats.archiveteam.org/wiki/Netscape_bookmarks), and isn't documented very well. This also means that Chrome (and probably Firefox) doesn't actually parse it as HTML, so the location of things like newlines actually matters.
